Declare missing function parameters. This fixes a crash on amd64, see
bug 215634. This patch has been applied upstream.

diff -Naur ./gtk/src/rbgtkiconview.c ../ruby-gnome2-all-0.16.0/gtk/src/rbgtkiconview.c
--- ./gtk/src/rbgtkiconview.c	2008-03-31 20:18:48.000000000 +0100
+++ ../ruby-gnome2-all-0.16.0/gtk/src/rbgtkiconview.c	2008-03-31 20:18:03.000000000 +0100
@@ -54,6 +54,8 @@
 static VALUE
 iview_get_path_at_pos(self, x, y)
     VALUE self;
+    VALUE x;
+    VALUE y;
 {
     GtkTreePath* path = gtk_icon_view_get_path_at_pos(_SELF(self),
                                                       NUM2INT(x), NUM2INT(y));
